doc/refman/RefMan-tac.tex

```diff
@@ -3684,11 +3709,12 @@ \subsubsection*{Creating Hint databases
   the number of subgoals generated by {\tt simple apply {\term}}.
 %{\tt auto} actually uses a slightly modified variant of {\tt simple apply} with use_metas_eagerly_in_conv_on_closed_terms set to false
 
-  % Is it really needed?
-  %% In case the inferred type of \term\ does not start with a product
-  %% the tactic added in the hint list is {\tt exact {\term}}. In case
-  %% this type can however be reduced to a type starting with a product,
-  %% the tactic {\tt apply {\term}} is also stored in the hints list.
+  In case the inferred type of \term\ does not start with a product
+  the tactic added in the hint list is {\tt exact {\term}}.
+% Actually, a slightly restricted version is used (no conversion on the head symbol)
+  In case
+  this type can however be reduced to a type starting with a product,
+  the tactic {\tt simple apply {\term}} is also stored in the hints list.
```

Comment:
This changed is a bit independent from the others but also strive to make the doc of `auto` more precise.
See also the discussion on 9227d6e.
Please someone, review this specifically (approve or reject or improve).